### PR TITLE
fix #76063 coinurl.net

### LINKED ISSUE #76063: coinurl.net
### Issue URL (Annoyance)
 [https://coinurl.net/SkinPack10](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fcoinurl.net%2FSkinPack10)
### Comment
Username: `@Bin312`
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> Frellwit's Swedish Filter

### PR DESCRIPTION
**Pull request by wout-codes**
#76063

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -9165,6 +9165,13 @@ brandbuffet.in.th#@#.category-advertorial
 !------------------------------ Vietnam ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/76063
+coinurl.net#%#//scriptlet("set-constant", "adsHeight", "15")
+coinurl.net#%#//scriptlet("prevent-window-open")
+coinurl.net##div > [href]
```
Comment (Alex-302) [inline EnglishFilter/sections/foreign.txt:9171]: dangerous rule.

File: EnglishFilter/sections/foreign.txt [hunk 2]
```diff
@@ -9165,6 +9165,13 @@ brandbuffet.in.th#@#.category-advertorial
...
+coinurl.net##a[href^="https://vnbit.org/"]
+!+ PLATFORM(ios, ext_safari, ext_android_cb)
+coinurl.net#@#[id*="ScriptRoot"]
```
Comment (Alex-302) [inline EnglishFilter/sections/foreign.txt:9174]: I guess `coinurl.net#%#//scriptlet("set-constant", "adsHeight", "15")` is not required with this exclusion.

File: EnglishFilter/sections/foreign.txt [hunk 3]
```diff
@@ -9165,6 +9165,13 @@ brandbuffet.in.th#@#.category-advertorial
 !------------------------------ Vietnam ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/76063
+coinurl.net#%#//scriptlet("set-constant", "adsHeight", "15")
+coinurl.net#%#//scriptlet("prevent-window-open")
+coinurl.net##div > [href]
+coinurl.net##a[href^="https://vnbit.org/"]
```
Comment (Alex-302) [inline EnglishFilter/sections/foreign.txt:9172]: better something like `://i.imgur.com/$domain=coinurl.net`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt:9174]: Required that only for ios, safari and cb. There is a leftover if I use it for adguard for windows.

File: EnglishFilter/sections/foreign.txt (hunk 3 above)
Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt:9172]: Added.

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt:9171]: I know it's risky but this is enough to block all banners without breaking anything.

**Pull request merged**